pkg/sslpolicy.go

```diff
@@ -11,12 +12,12 @@ import (
 
 // NewSslPolicy returns instance of the configuration options
 // necessary to create our globally enforced SSL Policy.
-func NewSslPolicy(name string) compute.SslPolicy {
+func NewSslPolicy(name string, profile string, version string) compute.SslPolicy {
```

Comment:
Variables has the same type can be grouped together
```suggestion
func NewSslPolicy(name, profile, version string) compute.SslPolicy {
```